### PR TITLE
Move test util files to testutils package #502

### LINKED ISSUE #502: Move test util files to testutils package
Some of the files in test/address/util should be moved to test/address/testuitls as the former is for test classes of util package and the latter is for utility classes used for testing.
At the same time, can rename testutils --> testutil to be consistent with util name (both singular).

### PR DESCRIPTION
**Pull request by yl-coder**
Fixes #502

### CODE REVIEW

**Comment by yl-coder** [comment]
Will do this after PR#491 is merged

File: src/test/java/address/keybindings/KeyBindingsManagerApiTest.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 
 import address.events.*;
-import address.util.TestUtil;
+import address.testutil.TestUtil;
```
Comment (damithc) [inline src/test/java/address/keybindings/KeyBindingsManagerApiTest.java]: We can put this in address.keybindings

File: src/test/java/address/TestApp.java [hunk 2]
```diff
@@ -8,7 +8,7 @@
 import address.sync.cloud.model.CloudAddressBook;
 import address.util.Config;
 import address.util.GuiSettings;
-import address.util.TestUtil;
+import address.testutil.TestUtil;
```
Comment (damithc) [inline src/test/java/address/TestApp.java]: Perhaps keep this in the same place MainApp is?

File: src/test/java/address/keybindings/KeyBindingsManagerApiTest.java (hunk 1 above)
Comment (damithc) [inline src/test/java/address/keybindings/KeyBindingsManagerApiTest.java]: Never mind. My mistake.

**Pull request merged**